doc/team_practices.md

```diff
@@ -0,0 +1,37 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
```

Comment:
> rationalize why a refactor is necessary

Beforehand? Most of my refactorings happen because I notice something as I'm working on something else. If they seem shorter than a couple hours and seem worthwhile (especially if they make the feature easier), I just go ahead and do them. Creating a story and waiting for it to be prioritized seems overkill, IMHO.

> track the effort involved

We usually categorize these as Chores in Tracker, and Chores don't get points by default :trollface: @annalee